### PR TITLE
make pending timeout customizable

### PR DESCRIPTION
**Pull request by cheyang**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1268)

### CODE REVIEW

File: components/arena/python/arena/_arena_distributed_tf_op.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def estimator_op(name, image, command,
                       evaluator=False, evaluator_cpu_limit=0, evaluator_memory_limit=0, 
                       env=[], data=[], sync_source=None,
                       metrics=['Train-accuracy:PERCENTAGE'],
-                      arena_image='cheyang/arena_launcher:v0.5',
+                      arena_image='cheyang/arena_launcher:v0.6',
```
Comment (animeshsingh) [inline components/arena/python/arena/_arena_distributed_tf_op.py]: Should these images be moved to a more neutral docker registry name?

File: components/arena/python/arena/_arena_distributed_tf_op.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def parameter_servers_op(name, image, command, env, data, sync_source, annotatio
                       tensorboard, 
                       worker_port, ps_port, 
                       metrics=['Train-accuracy:PERCENTAGE'],
-                      arena_image='cheyang/arena_launcher:v0.5',
+                      arena_image='cheyang/arena_launcher:v0.6',
```
Comment (animeshsingh) [inline components/arena/python/arena/_arena_distributed_tf_op.py]: See above

File: components/arena/docker/arena_launcher.py [hunk 3]
```diff
@@ -250,6 +250,9 @@ def main(argv=None):
   parser.add_argument('--timeout-hours', type=int,
                       default=200,
                       help='Time in hours to wait for the Job submitted by arena to complete')
+  parser.add_argument('--pending-timeout-minutes', type=int,
+                      default=360,
+                      help='Time in hours to wait for the Job submitted by arena from pending to running')
```
Comment (animeshsingh) [inline components/arena/docker/arena_launcher.py]: The var says --pending-timeout-minutes, whereas the descriptions says hours?

Comment (cheyang) [reply components/arena/docker/arena_launcher.py]: Thank you for spotting it！ Fix it.

File: components/arena/python/arena/_arena_distributed_tf_op.py (hunk 1 above)
Comment (cheyang) [reply components/arena/python/arena/_arena_distributed_tf_op.py]: I agree with your suggestions. I'd like to move it to a neutral docker registry name. I will keep it in mind and try to find. Thanks.

**Comment by animeshsingh** [comment]
Thanks for the changes.
/lgtm
/approve

File: components/arena/docker/arena_launcher.py [hunk 4]
```diff
@@ -249,7 +249,10 @@ def main(argv=None):
   parser.add_argument('--tensorboard-image', type=str, default='tensorflow/tensorflow:1.12.0')
   parser.add_argument('--timeout-hours', type=int,
                       default=200,
-                      help='Time in hours to wait for the Job submitted by arena to complete')
+                      help='Time in minutes to wait for the Job submitted by arena to complete')
```
Comment (hongye-sun) [inline components/arena/docker/arena_launcher.py]: It should be hours, right?

**Comment by hongye-sun** [comment]
/approve
/hold

Please unhold the PR when you feel it's ready to merge. Thanks.

File: components/arena/docker/arena_launcher.py (hunk 4 above)
Comment (cheyang) [reply components/arena/docker/arena_launcher.py]: Yes, because training always takes more than 1 hour.

**Pull request merged**